pages/api/auth/[...nextauth].page.ts

```diff
@@ -272,9 +272,7 @@ const Auth = (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
           return url;
         }
         if (url === 'signOut' && AUTH_PROVIDER === 'OKTA') {
-          return `https://signon.okta.com/login/signout?fromURI=${encodeURIComponent(
-            process.env.OKTA_SIGNOUT_REDIRECT_URL ?? '',
-          )}`;
+          return `https://signon.okta.com/login/signout?fromURI=${process.env.SITE_URL}/login`;
```

Comment:
Why are we dropping the URI encoding?

Reply:
Ah, that was a mistake

Reply:
Actually, I guess we could leave`OKTA_SIGNOUT_REDIRECT_URL` to allow the non-Cru orgs flexibility to send the user wherever they need after they log out.